packages/components/Menu/src/MenuItemRadio/index.ts

```diff
@@ -1,2 +1,9 @@
 export { MenuItemRadio, menuItemRadioName } from './MenuItemRadio';
 export { useMenuItemRadio } from './useMenuItemRadio';
+export type {
+  MenuItemRadioProps,
```

Comment:
I don't think this is going to work. The exported component has a different type signature based on the platform, I think that's going to cause problems downstream

Reply:
Removed. 
But we had to move tokens to a common place to CheckboxTokens as the common tsx file imports it from CheckboxTokens types. 

MenuItemRadio.customize({}) doesn't give types suggestions for tokens.